CHANGELOG.md

```diff
@@ -2,6 +2,14 @@
 
 All notable changes to this project will be documented in this file. This project make usage of the [Yii Versioning Strategy](https://github.com/yiisoft/yii2/blob/master/docs/internals/versions.md). In order to read more about upgrading and BC breaks have a look at the [UPGRADE Document](UPGRADE.md).
 
+## 1.0.1 (in progress)
+
+- Update lickstarter style a little bit in general
```

Comment:
the changelog should refer to a pr or issue see an example of how we do this in general: https://github.com/luyadev/luya/blob/master/CHANGELOG.md